src/api/public/apidocs-new/paths/source_project_name_package_name_cmd_diff.yaml

```diff
@@ -0,0 +1,173 @@
+post:
+  summary: Returns the source diff of a package.
+  description: |
+    Without parameters, return the diff of the last change made to the package. This command doesn't perform any action.
+  security:
+    - basic_authentication: []
+  parameters:
+    - $ref: '../components/parameters/project_name.yaml'
+    - $ref: '../components/parameters/package_name.yaml'
+    - in: query
+      name: cacheonly
+      schema:
+        type: string
+      description: |
+        Set to `1` to retrieve a diff only if the diff is already cached.
+
+        If the diff is not cached return a 412 "Precondition failed" error.
+      example: 1
+    - in: query
+      name: expand
+      schema:
+        type: string
+      description: Set to `1` to expand any link before diffing.
+      example: 1
+    - in: query
+      name: file
+      schema:
+        type: string
+      description: Limit diff to the given file name.
+      example: hello_world.spec
+    - in: query
+      name: filelimit
+      schema:
+        type: string
+      description: |
+        Limit the diff size by line numbers.
+
+        By default it takes a built-in default value.
+
+        Set to `0` to disable the limit.
+      example: 20
+    - in: query
+      name: linkrev
+      schema:
+        type: string
+      description: Link revision in base package. Set to `base` to use the commit revision.
+      example: base
+    - in: query
+      name: meta
+      schema:
+        type: string
+      description: Set to `1` to diff meta data instead of sources.
+      example: 1
+    - in: query
+      name: missingok
+      schema:
+        type: string
+      description: Set to `1` to diff against an inexistent origin.
+      example: 1
+    - in: query
+      name: nodiff
+      schema:
+        type: string
+      description: Set to `1` to only show changed files, and not details inside files.
+      example:
+    - in: query
+      name: olinkrev
+      schema:
+        type: string
+      description: Link revision of the origin package.
+    - in: query
+      name: onlyissues
+      schema:
+        type: string
+      description: Set to `1` to show no source diff, only the tracker issues and their change state.
+      example: 1
+    - in: query
+      name: orev
+      schema:
+        type: string
+      description: Old revision.
+    - in: query
+      name: opackage
+      schema:
+        type: string
+      description: Old package.
+    - in: query
+      name: oproject
+      schema:
+        type: string
+      description: Old project.
+    - in: query
+      name: rev
+      schema:
+        type: string
+      description: Revision of the package.
+    - in: query
+      name: tarlimit
+      schema:
+        type: string
+      description: Limit the diff size by line numbers.
+      example: 20
+    - in: query
+      name: unified
+      schema:
+        type: string
+      description: Set to `1` to use unified format without looking into archives.
```

Comment:
What are archives?

Reply:
I rephrased the description. Please have a look.